Redesign

Waakul opened the pull request:
## Note
Please don't judge through screenshots, the screenshots are completely color and quality corrupted, Please try the pr to see the changes instead.

Discussion #7215 

### Changes
- Recolor for both dark and light modes
- light Box-shadows instead of borders
- Expand width on focusing the search bar in the settings page
- Nicer looking tabs
- Tab's width adjust to fill availible space
- Center aligned UI instead of left alignment
- Improved shadows
- A very very few bug fixes from the previous UI (I forgot what they are)
- Things that are supposed to be in containers that look bad when not now in containers
- A puzzle icon instead of the settings icon cause it's more meaningful
- Consistent Design Language
- A ton of minor redesigns i forgot, check changes ig

### Reason for changes
Overall Modern Look Without Ruining Accessibility

### Tests
Tested In: Microsoft Edge, Firefox

DNin01 commented:
Could we see a screenshot, maybe?

Waakul commented:
> Could we see a screenshot, maybe?

dark mode or light mode?

Waakul commented:
Can anyone beta-test it and provide suggestions?

Samq64 commented:
Could the tabs expand to fill the available space? Most tabs are toggleable leaving this when they're disabled:

![image](https://github.com/ScratchAddons/ScratchAddons/assets/81489795/076803ae-d29a-40b2-a670-6e543411ddd2)

Samq64 commented on webpages/popup/style.css:
```diff
@@ -60,6 +60,8 @@ body {
   padding-bottom: 0px;
   display: flex;
   border-bottom: 1px solid var(--control-border);
+  display: flex;
```
This is already `display: flex`.
```suggestion
```

Samq64 commented on webpages/styles/components/buttons.css:
```diff
@@ -1,7 +1,7 @@
   color: var(--content-text);
   background: var(--button-background);
-  border: 1px solid var(--control-border);
+  border: none;
```
I think the border should stay on the buttons.

Waakul replied:
the box shadow works as the border

Samq64 replied:
There are still some places where buttons are directly on the page background which causes poor contrast but now also without a border.

Samq64 replied:
Also a couple buttons used to be highlighted yellow such as the confirm button when importing settings.

Samq64 commented on webpages/styles/components/search.css:
```diff
@@ -1,12 +1,10 @@
 .search-box {
-  background: var(--search-toolbar-background);
```
I'm not so sure about this color change Since it doesn't scroll the same way as he background.

Waakul replied:
idk i didnt add the color change, git is getting confused ig

Waakul replied:
also there is a box-shadow to differentiate

Samq64 commented:
If you're looking for more ideas you could look at what would be pretty simple to implement from [this search filter](https://github.com/ScratchAddons/ScratchAddons/issues?q=is%3Aopen+is%3Aissue+label%3A%22scope%3A+webpages%22+label%3A%22scope%3A+design%22+).

Waakul commented:
![image](https://github.com/ScratchAddons/ScratchAddons/assets/135821321/876875a9-e9cc-45e0-94f3-bbe964daef4b)
settings page

Waakul commented:
> Could the tabs expand to fill the available space? Most tabs are toggleable leaving this when they're disabled:
> 
> ![image](https://private-user-images.githubusercontent.com/81489795/307465892-076803ae-d29a-40b2-a670-6e543411ddd2.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDg3NjUzODIsIm5iZiI6MTcwODc2NTA4MiwicGF0aCI6Ii84MTQ4OTc5NS8zMDc0NjU4OTItMDc2ODAzYWUtZDI5YS00MGIyLWE2NzAtNmU1NDM0MTFkZGQyLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAyMjQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMjI0VDA4NTgwMlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWQ1MjBkMTcxYjhhNTM0ZWJlYjg1YjkyMDIzYjUxMjcwYTZlZjcyMGQ1Y2ViNDRjMzQyZDBiNzVkNjQ1Yjk0NmUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.K6IQVIheB2ELCtpQLQd1FGfqrxxtdgc7asAxSpRUGcE)

good idea!

Waakul commented:
> Could the tabs expand to fill the available space? Most tabs are toggleable leaving this when they're disabled:
> 
> ![image](https://private-user-images.githubusercontent.com/81489795/307465892-076803ae-d29a-40b2-a670-6e543411ddd2.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDg3Nzk3NjcsIm5iZiI6MTcwODc3OTQ2NywicGF0aCI6Ii84MTQ4OTc5NS8zMDc0NjU4OTItMDc2ODAzYWUtZDI5YS00MGIyLWE2NzAtNmU1NDM0MTFkZGQyLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAyMjQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMjI0VDEyNTc0N1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTczOWYxMzQ4ZWFkNzNjZmFjYTU3YWI2MTVhNWJjNTdiMjEwMTVmNDVlOThhYTkxMTYyMzVkYTAwYTA4NDcwYWUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.NdtzB5B34bw3LUjXN4MqhyftFyz4IMbXuWhkZTsZcGk)

pushed it :)

Waakul commented:
![image](https://github.com/ScratchAddons/ScratchAddons/assets/135821321/2f044ae0-229d-4f97-99c8-80c96f5fe006)
daily updates

Samq64 commented on popups/scratch-messaging/popup.css:
```diff
@@ -270,16 +272,20 @@ a.message-type-title-text {
 .status-container {
-  padding-top: 10px;
-  padding-bottom: 20px;
+  padding-top: 15px;
+  padding-bottom: 15px;
```
```suggestion
  padding: 15px 0;
```
I would also decrease the padding here a little too but that's not really important here.

Samq64 commented on webpages/settings/components/addon-body.html:
```diff
@@ -263,12 +262,12 @@
   }
   .addon-message {
     padding: 8px;
-    border: 2px solid;
```
Without the border the yellow notices are very hard to distinguish from the normal ones without the border. Could `--notice-background` be changed to yellow in dark mode?

Samq64 commented on webpages/settings/components/addon-body.html:
```diff
@@ -311,6 +309,7 @@
   }
   .addon-settings {
     padding-top: 5px;
+    padding-bottom: 5px;
```
```suggestion
    padding: 5px 0;
```

Samq64 commented on webpages/settings/components/addon-body.html:
```diff
@@ -369,8 +368,8 @@
     position: relative;
     cursor: pointer;
     transition: all 0.25s ease;
-    border: 1px solid var(--switch-border);
```
Changing the button's colour and removing the border has unintentionally reverted the switch part of #6240.

Samq64 commented on webpages/styles/colors-light.css:
There are probably a couple unused border colours in these two files now.

Samq64 commented on webpages/styles/components/search.css:
```diff
@@ -15,6 +14,10 @@
   color: var(--content-text);
   width: 150px;
   height: 16px;
+  transition: none;
```
What was wrong with the transition?

Samq64 reviewed (commented):
Overall I like the design changes but without good contrast removing borders can be a problem.

Samq64 commented:
Also could you add some screenshots to the original post of some individual complainants? Looking at everything as a whole it's not that obvious what changed besides the tabs.

Samq64 commented on webpages/settings/components/category-selector.html:
```diff
@@ -68,8 +68,7 @@
-    border-radius: 0 4px 4px 0;
-    height: 40px;
+    height: 100%;
     background: var(--orange);
```
This could also be changed to `--brand-orange` now that we don't have to worry about icon-recoloring (#6240).

Waakul replied:
just got unbanned, also i dont wanna make @redguy12 kill me

Waakul replied:
> just got unbanned, also i dont wanna make @RedGuy12 kill me

ask redguy12 why he'll never stop yapping at me if i agree with this suggestion

Secret-chest commented:
While we're at it, remove Sora.

Secret-chest commented:
Also, why make the settings a puzzle?

BroJac5246 commented:
> While we're at it, remove Sora.

Lol it's been so long since that's been discussed that I thought you were talking about the AI video generation model 😂

Secret-chest commented:
In Romanian "sora" means "the sister", so it's ChatGPT's sister

Samq64 commented:
> While we're at it, remove Sora.

#4656 is still open so it should probably stay separate for now.

Waakul commented:
I iwll share some screenshots soon

Waakul commented:
This redesign makes it have a modern fell rather than adding new features

Secret-chest commented:
Can you merge the tabs into the top bar like this?

![Tabs](https://i.postimg.cc/pX5YknWQ/Screenshot-20240304-122812-Octo-Droid.png)

Waakul commented:
> Can you merge the tabs into the top bar like this?
> 
> ![Tabs](https://camo.githubusercontent.com/aedd1a08aff3f708a04920f6f8e3742aea47438c2db6e563ff99e101123a6882/68747470733a2f2f692e706f7374696d672e63632f705835596b6e57512f53637265656e73686f742d32303234303330342d3132323831322d4f63746f2d44726f69642e706e67)

hmm

Waakul commented:
> Can you merge the tabs into the top bar like this?
> 
> ![Tabs](https://camo.githubusercontent.com/aedd1a08aff3f708a04920f6f8e3742aea47438c2db6e563ff99e101123a6882/68747470733a2f2f692e706f7374696d672e63632f705835596b6e57512f53637265656e73686f742d32303234303330342d3132323831322d4f63746f2d44726f69642e706e67)

![image](https://github.com/ScratchAddons/ScratchAddons/assets/135821321/c988f132-b5c8-47e1-b8d6-dc926a6e26a2)
nah, it looks bad

BroJac5246 commented:
> > Can you merge the tabs into the top bar like this?
> > 
> > ![Tabs](https://camo.githubusercontent.com/aedd1a08aff3f708a04920f6f8e3742aea47438c2db6e563ff99e101123a6882/68747470733a2f2f692e706f7374696d672e63632f705835596b6e57512f53637265656e73686f742d32303234303330342d3132323831322d4f63746f2d44726f69642e706e67)
> 
> -snip-
> nah, it looks bad

Can you send a screenshot of what it currently looks like?

Waakul commented:
![image](https://github.com/ScratchAddons/ScratchAddons/assets/135821321/250dcb52-90f9-43e9-9c22-eff81f644543)
slight shadow changes, more accessible?

DNin01 commented:
> Can you merge the tabs into the top bar like this?

IMO, it makes it look cleaner but also wrong… Maybe it has to do with how important the Scratch Addons top bar is.

An in-between might be to blend it in more (just by darkening the top bar in dark mode).

Secret-chest commented:
Cog is universally recognised symbol for settings. Revert.

DNin01 commented on webpages/settings/components/addon-setting.html:
Without a border, the switches, when set to off, are very hard to make out against the background. One way to solve this would be to lighten the switch background in dark mode.

Waakul replied:
i would prefer darkening it

DNin01 replied:
A border would probably be the best option, then. More designers are starting to follow the "outlined means inactive and filled in means active" approach.

Don't worry, it's not like everything has to not have a border.

DNin01 commented on webpages/settings/components/addon-setting.html:
The switch handles should also be offset 5px instead of 4px relative to the corner. Otherwise, they look misplaced. This was caused by the border being removed.

Basically, wherever you see `top: 4px`, `left: 4px`, or `left: 24px` within style rules selecting `::before`, change those to multiples of 5.

Waakul replied:
fixed

DNin01 commented on images/icons/settings.svg:
I don't think there's any problems with the cog icon.

Waakul commented:
> Cog is universally recognised symbol for settings. Revert.

Yes, but it is not settings. it's full-screen addons page

DNin01 commented:
> > Cog is universally recognised symbol for settings. Revert.
> 
> Yes, but it is not settings. it's full-screen addons page

👍 and 👎

Yeah, I don't really know what to do about that 🤣

Waakul commented:
> > > Cog is universally recognised symbol for settings. Revert.
> > 
> > 
> > Yes, but it is not settings. it's full-screen addons page
> 
> 👍 and 👎
> 
> Yeah, I don't really know what to do about that 🤣

if it was settings then it should open this menu instead 😕 :
![image](https://github.com/ScratchAddons/ScratchAddons/assets/135821321/4f5ace52-74c8-454a-9d61-429c9835c6ca)

Waakul commented:
![Frame 1 (17)](https://github.com/ScratchAddons/ScratchAddons/assets/135821321/e145a462-e6f8-4a62-81c1-6aa6cb0c6345)
A full comparison, should i share the fullscreen page too?

BroJac5246 commented:
> > Cog is universally recognised symbol for settings. Revert.
> 
> Yes, but it is not settings. it's full-screen addons page

Aka a settings page. Just use the cog, it's more recognizable.

Waakul commented:
> > > Cog is universally recognised symbol for settings. Revert.
> > 
> > 
> > Yes, but it is not settings. it's full-screen addons page
> 
> Aka a settings page. Just use the cog, it's more recognizable.

no, settings would mean scratchaddons's settings but it's not

BroJac5246 commented:
> > > > Cog is universally recognised symbol for settings. Revert.
> > > 
> > > 
> > > Yes, but it is not settings. it's full-screen addons page
> > 
> > Aka a settings page. Just use the cog, it's more recognizable.
> 
> no, settings would mean scratchaddons's settings but it's not

Are the addons not part of Scratch Addons? Of course they are. The toggles are settings and enabling them enables part of Scratch Addons, so they're Scratch Addons settings. The 'More settings' popup simply includes _additional_ settings (hence the name!).

It occurred to me that an 'Open in new tab/window' icon would be more intuitive than the current solutions, however, since the current buttons just look like icons.

Waakul commented:
> > > > > Cog is universally recognised symbol for settings. Revert.
> > > > 
> > > > 
> > > > Yes, but it is not settings. it's full-screen addons page
> > > 
> > > 
> > > Aka a settings page. Just use the cog, it's more recognizable.
> > 
> > 
> > no, settings would mean scratchaddons's settings but it's not
> 
> Are the addons not part of Scratch Addons? Of course they are. The toggles are settings and enabling them enables part of Scratch Addons, so they're Scratch Addons settings. The 'More settings' popup simply includes _additional_ settings (hence the name!).
> 
> It occurred to me that an 'Open in new tab/window' icon would be more intuitive than the current solutions, however, since the current buttons just look like icons.

sorry but no, that looks bad fr

BroJac5246 commented:
> > > > > > Cog is universally recognised symbol for settings. Revert.
> > > > > 
> > > > > 
> > > > > Yes, but it is not settings. it's full-screen addons page
> > > > 
> > > > 
> > > > Aka a settings page. Just use the cog, it's more recognizable.
> > > 
> > > 
> > > no, settings would mean scratchaddons's settings but it's not
> > 
> > Are the addons not part of Scratch Addons? Of course they are. The toggles are settings and enabling them enables part of Scratch Addons, so they're Scratch Addons settings. The 'More settings' popup simply includes _additional_ settings (hence the name!).
> > 
> > It occurred to me that an 'Open in new tab/window' icon would be more intuitive than the current solutions, however, since the current buttons just look like icons.
> 
> sorry but no, that looks bad fr

@DNin01 what is your take?

DNin01 commented:
> @DNin01 what is your take?

At the end of the day, I'd probably choose the cog icon because it's easier to tell what it does. If that's not enough, we do still have the "explore all n+ addons" notification to guide the user in the right direction.

Also, the hovernote for the button should match its purpose - if we keep the bulleted list icon, it could have a `title` of "All Addons" or something.

Secret-chest commented:
> sorry but no, that looks bad fr

Stop talking like a teenager.

The addons page is Scratch Addons' settings. It lists you what features you can enable, disable and customise.

If we don't use a cog, we would need to have one of those circles pointing to the icon like Google apps.

Waakul commented:
> > sorry but no, that looks bad fr
> 
> Stop talking like a teenager.
> 
> The addons page is Scratch Addons' settings. It lists you what features you can enable, disable and customise.
> 
> If we don't use a cog, we would need to have one of those circles pointing to the icon on first use like Google apps.

scratch addons's settings are these:
![image](https://github.com/ScratchAddons/ScratchAddons/assets/135821321/a108d104-654f-45a6-bf5e-07285e5c0f83)

Waakul commented:
> > sorry but no, that looks bad fr
> 
> Stop talking like a teenager.
> 
> The addons page is Scratch Addons' settings. It lists you what features you can enable, disable and customise.
> 
> If we don't use a cog, we would need to have one of those circles pointing to the icon on first use like Google apps.

I'm talking like a teen cause i am one (duh).

CubesterYT commented:
> > > sorry but no, that looks bad fr
> > 
> > Stop talking like a teenager.
> > 
> > The addons page is Scratch Addons' settings. It lists you what features you can enable, disable and customise.
> > 
> > If we don't use a cog, we would need to have one of those circles pointing to the icon on first use like Google apps.
> 
> I'm talking like a teen cause i am one (duh).

Ok? Just because you are a teenager, doesn't mean you should in a serious and professional environment. Most of us here are teens as well, who also have lives, but we're dedicated professionally to Scratch Addons, this isn't a place to fool around. If you can't handle that, you probably shouldn't make situations worse, please talk in a more serious voice.

Waakul commented:
![Redesign by Waakul · Pull Request #7214 · ScratchAddons_ScratchAddons and 4 more pages - Personal - Microsoft​ Edge 3_9_2024 2_32_14 PM](https://github.com/ScratchAddons/ScratchAddons/assets/135821321/947323b9-25dd-4b93-aa7c-66f6262b4f4f)
![Redesign by Waakul · Pull Request #7214 · ScratchAddons_ScratchAddons and 4 more pages - Personal - Microsoft​ Edge 3_9_2024 2_32_25 PM](https://github.com/ScratchAddons/ScratchAddons/assets/135821321/c5110605-09bb-45af-8759-0a596b72a391)
![Redesign by Waakul · Pull Request #7214 · ScratchAddons_ScratchAddons and 4 more pages - Personal - Microsoft​ Edge 3_9_2024 2_32_37 PM](https://github.com/ScratchAddons/ScratchAddons/assets/135821321/7dca6547-3af3-44d5-a1f3-903ff122ab68)
Current screenshots

Waakul commented:
The aim of this redesign was to make it modern-looking and simple. I guess it's time to mark it as ready for review

Waakul commented:
![Screenshot 2024-03-09 144213](https://github.com/ScratchAddons/ScratchAddons/assets/135821321/15c2ca14-0231-45e3-b6a3-1fdbc0cfa3f5)
![Screenshot 2024-03-09 144218](https://github.com/ScratchAddons/ScratchAddons/assets/135821321/00331542-179e-4258-926d-ecef9cb85517)
![Screenshot 2024-03-09 144222](https://github.com/ScratchAddons/ScratchAddons/assets/135821321/ba6d780d-838c-4254-90b2-1fe9bc51b344)
![Screenshot 2024-03-09 144236](https://github.com/ScratchAddons/ScratchAddons/assets/135821321/b91c0302-50f2-41e2-b9f8-60bb5470a3b0)
Screenshots of some other stuff

Waakul commented:
I've implemented some design bug fixes too

Waakul commented:
### ALL ABOVE IMAGES ARE OUTDATED, EVEN THE ONES I SEND TODAY (I ADDED MORE CONTRAST)

Samq64 commented on webpages/styles/colors.css:
```diff
@@ -18,19 +18,21 @@
+  --comment-background: #1c1c1c;
+  --unread-comment-background: #0d0d0d;
+  --unread-comment-border: inset 0px 0px 0px 2px #161616;
   --notice-background: var(--navigation-background);
```
Could this be made yellow? It's hard to tell the difference between normal and yellow notices right now.

Waakul replied:
wdym by "yellow notices" ?

Samq64 replied:
Stable: 
![image](https://github.com/ScratchAddons/ScratchAddons/assets/81489795/87dd331e-f355-43b1-aa7b-964b6dce799d)
PR:
![image](https://github.com/ScratchAddons/ScratchAddons/assets/81489795/4402053d-71cc-4897-97ba-4bd295da170a)

Waakul replied:
> Stable: ![image](https://private-user-images.githubusercontent.com/81489795/311513770-87dd331e-f355-43b1-aa7b-964b6dce799d.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTAwNzgyNTYsIm5iZiI6MTcxMDA3Nzk1NiwicGF0aCI6Ii84MTQ4OTc5NS8zMTE1MTM3NzAtODdkZDMzMWUtZjM1NS00M2IxLWFhN2ItOTY0YjZkY2U3OTlkLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMTAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzEwVDEzMzkxNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTViYjI1MTA3NmM3ZTE3MjJiZTAxMmE1NmViNDZlMThlNTI1ZDFmNjZkODYyZDc0ZjY2MzA1OWU5ZGI1ZmYxZmImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.zWCVWqM9BBYQ-mazo_lI1dy4sVdyDECtoUNx2syxDTc) PR: ![image](https://private-user-images.githubusercontent.com/81489795/311513785-4402053d-71cc-4897-97ba-4bd295da170a.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTAwNzgyNTYsIm5iZiI6MTcxMDA3Nzk1NiwicGF0aCI6Ii84MTQ4OTc5NS8zMTE1MTM3ODUtNDQwMjA1M2QtNzFjYy00ODk3LTk3YmEtNGJkMjk1ZGExNzBhLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMTAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzEwVDEzMzkxNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTAwZmEwNGY2NzkxODJlMjE4OTdlMDA3YTRmOGEyODM4NGRhZTgyODhhY2VjZWYyNjg3NmFmMDYzMjc4ZTEzNjQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.Y148jv-GlVhlzZM1JInCSPcXHCKAsL_F7o2VNo4rAXc)
> 
> It already has a yellow background in light mode.

Oh

Waakul replied:
Just pushed it :)

Samq64 commented on webpages/styles/colors.css:
```diff
@@ -41,18 +43,20 @@
   --info-border: var(--content-separator);
   --update-background: var(--navigation-background);
-  --update-border: var(--purple-opacity75);
+  --update-bo18rder: var(--purple-opacity75);
```
Accident?

Waakul replied:
woops

Waakul commented:
@Samq64 any suggestions? are you gonna approve?

Waakul commented:
![Extensions and 11 more pages - Personal - Microsoft​ Edge 3_10_2024 4_47_36 PM](https://github.com/ScratchAddons/ScratchAddons/assets/135821321/a98aba37-828b-4065-90a5-f623b8846cca)
Updates ig

Samq64 commented:
There are still a couple places that used to highlight borders in certain cases:
![image](https://github.com/ScratchAddons/ScratchAddons/assets/81489795/4f4b5652-a315-41e5-b7d2-a4947fd69bd9)
![image](https://github.com/ScratchAddons/ScratchAddons/assets/81489795/e9bc68ef-b64d-4999-8639-463b5b6a0af9)
I'm no sure if there are others.

Other than that I think it looks pretty good.

Waakul commented:
> There are still a couple places that used to highlight borders in certain cases: ![image](https://private-user-images.githubusercontent.com/81489795/311514548-4f4b5652-a315-41e5-b7d2-a4947fd69bd9.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTAwNzkwNTgsIm5iZiI6MTcxMDA3ODc1OCwicGF0aCI6Ii84MTQ4OTc5NS8zMTE1MTQ1NDgtNGY0YjU2NTItYTMxNS00MWU1LWI3ZDItYTQ5NDdmZDY5YmQ5LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMTAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzEwVDEzNTIzOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWFlZjRhMWEyMWI4Y2YwZDgwOGQzZWZmYTJhZWRhN2ZiOTU1ZWRiZjk0MmZkNjA4N2JjZDE0NzVjNGRjNDA0OWEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.4_v0vtu_unK8oND3ECWd5nhiYb4_OOhocAMZWD2y5LA) ![image](https://private-user-images.githubusercontent.com/81489795/311514586-e9bc68ef-b64d-4999-8639-463b5b6a0af9.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTAwNzkwNTgsIm5iZiI6MTcxMDA3ODc1OCwicGF0aCI6Ii84MTQ4OTc5NS8zMTE1MTQ1ODYtZTliYzY4ZWYtYjY0ZC00OTk5LTg2MzktNDYzYjViNmEwYWY5LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMTAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzEwVDEzNTIzOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTk4ZTJkNWI1MWFhMzNiODdjODI4OGQzODAwODNjNjA3NmYzN2M0NjU5Mzc4ZjdiZWU0MGE1MTI5MWIxZTEyZjkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.Rmi_ET3HU8SM6Lb1cTl_XJrVQLx3SsV8Aggmkz6pREg) I'm no sure if there are others.
> 
> Other than that I think it looks pretty good.

Hmm, ok will fix it now

Waakul commented:
> There are still a couple places that used to highlight borders in certain cases: ![image](https://private-user-images.githubusercontent.com/81489795/311514548-4f4b5652-a315-41e5-b7d2-a4947fd69bd9.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTAwNzkwNTgsIm5iZiI6MTcxMDA3ODc1OCwicGF0aCI6Ii84MTQ4OTc5NS8zMTE1MTQ1NDgtNGY0YjU2NTItYTMxNS00MWU1LWI3ZDItYTQ5NDdmZDY5YmQ5LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMTAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzEwVDEzNTIzOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWFlZjRhMWEyMWI4Y2YwZDgwOGQzZWZmYTJhZWRhN2ZiOTU1ZWRiZjk0MmZkNjA4N2JjZDE0NzVjNGRjNDA0OWEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.4_v0vtu_unK8oND3ECWd5nhiYb4_OOhocAMZWD2y5LA) ![image](https://private-user-images.githubusercontent.com/81489795/311514586-e9bc68ef-b64d-4999-8639-463b5b6a0af9.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTAwNzkwNTgsIm5iZiI6MTcxMDA3ODc1OCwicGF0aCI6Ii84MTQ4OTc5NS8zMTE1MTQ1ODYtZTliYzY4ZWYtYjY0ZC00OTk5LTg2MzktNDYzYjViNmEwYWY5LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMTAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzEwVDEzNTIzOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTk4ZTJkNWI1MWFhMzNiODdjODI4OGQzODAwODNjNjA3NmYzN2M0NjU5Mzc4ZjdiZWU0MGE1MTI5MWIxZTEyZjkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.Rmi_ET3HU8SM6Lb1cTl_XJrVQLx3SsV8Aggmkz6pREg) I'm no sure if there are others.
> 
> Other than that I think it looks pretty good.

i think that i should use brand orange for the confirm thingy

Waakul commented:
how am i supposed to test the alert?, i cant just break the cg or smth

Waakul commented:
> how am i supposed to test the alert?, i cant just break the cg or smth:
![image](https://github.com/ScratchAddons/ScratchAddons/assets/135821321/62dfc4ca-9bf1-4d9a-af6c-663ae4ff137d)
did it in my test 💀, will i get banned or smth or it?

Waakul commented:
> There are still a couple places that used to highlight borders in certain cases: ![image](https://private-user-images.githubusercontent.com/81489795/311514548-4f4b5652-a315-41e5-b7d2-a4947fd69bd9.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTAwNzkwNTgsIm5iZiI6MTcxMDA3ODc1OCwicGF0aCI6Ii84MTQ4OTc5NS8zMTE1MTQ1NDgtNGY0YjU2NTItYTMxNS00MWU1LWI3ZDItYTQ5NDdmZDY5YmQ5LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMTAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzEwVDEzNTIzOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWFlZjRhMWEyMWI4Y2YwZDgwOGQzZWZmYTJhZWRhN2ZiOTU1ZWRiZjk0MmZkNjA4N2JjZDE0NzVjNGRjNDA0OWEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.4_v0vtu_unK8oND3ECWd5nhiYb4_OOhocAMZWD2y5LA) ![image](https://private-user-images.githubusercontent.com/81489795/311514586-e9bc68ef-b64d-4999-8639-463b5b6a0af9.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTAwNzkwNTgsIm5iZiI6MTcxMDA3ODc1OCwicGF0aCI6Ii84MTQ4OTc5NS8zMTE1MTQ1ODYtZTliYzY4ZWYtYjY0ZC00OTk5LTg2MzktNDYzYjViNmEwYWY5LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMTAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzEwVDEzNTIzOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTk4ZTJkNWI1MWFhMzNiODdjODI4OGQzODAwODNjNjA3NmYzN2M0NjU5Mzc4ZjdiZWU0MGE1MTI5MWIxZTEyZjkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.Rmi_ET3HU8SM6Lb1cTl_XJrVQLx3SsV8Aggmkz6pREg) I'm no sure if there are others.
> 
> Other than that I think it looks pretty good.

just pushed those fixes too, now approve?

Waakul commented:
I was testing another pr and it looked so weird when i was using my UI for almost 1 month and suddenly switch back to the old UI XD.

Waakul commented:
I JUST RENAMED THE BRANCH AND STUPID GITHUB CLOSED THE PR

MaterArc commented:
> I JUST RENAMED THE BRANCH AND STUPID GITHUB CLOSED THE PR

That's how GitHub works, it was the head branch...

Waakul commented:
> > I JUST RENAMED THE BRANCH AND STUPID GITHUB CLOSED THE PR
> 
> That's how GitHub works, it was the head branch...

i didnt delete it or smth, i just renamed it

Waakul commented:
> > > I JUST RENAMED THE BRANCH AND STUPID GITHUB CLOSED THE PR
> > 
> > 
> > That's how GitHub works, it was the head branch...
> 
> i didnt delete it or smth, i just renamed it (so why cant github rename all the mentions of this branch "on" the website)

Waakul commented:
It's better to merge this before vue3 gets merged so that i dont need to deal with all the merge conflicts (i suck at git)

Samq64 commented on webpages/styles/colors.css:
```diff
@@ -18,20 +18,23 @@
+  --admin-comment-background: #0000007a;
+  --unread-comment-background: #0d0d0d;
+  --unread-comment-border: inset 0px 0px 0px 2px #161616;
+  --notice-background: var(--yellow-opacity50);
```
This doesn't look great on the dark background.
![image](https://github.com/ScratchAddons/ScratchAddons/assets/81489795/02d2e350-3687-414d-bf3e-f358c37fa904)

Waakul replied:
hmm

Waakul replied:
![image](https://github.com/ScratchAddons/ScratchAddons/assets/135821321/7691d9b8-c3de-43f1-a9b6-2739006a6ef8)
wbu this?

Samq64 replied:
That's fine unless you want to just use `--yellow`.

Samq64 commented:
> It's better to merge this before vue3 gets merged so that i dont need to deal with all the merge conflicts (i suck at git)

We have write access to this branch.

Waakul commented:
> > It's better to merge this before vue3 gets merged so that i dont need to deal with all the merge conflicts (i suck at git)
> 
> Org members have write access to this branch.

ik

Samq64 commented on popups/scratch-messaging/popup.css:
```diff
@@ -30,25 +30,29 @@ button {
 }
 .message-type-admin {
-  border-color: var(--orange);
+  background: var(--yellow-opacity50);
```
```suggestion
  background: var(--notice-background);
```

Waakul replied:
I forgot about that sry

Samq64 commented:
One last thing, these shadows look weird.
![image](https://github.com/ScratchAddons/ScratchAddons/assets/81489795/cf94602b-fe0a-4061-8b5f-42228f35ca2b)

Waakul commented:
> One last thing, these shadows look weird. ![image](https://private-user-images.githubusercontent.com/81489795/311535187-cf94602b-fe0a-4061-8b5f-42228f35ca2b.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTAwOTUzMTIsIm5iZiI6MTcxMDA5NTAxMiwicGF0aCI6Ii84MTQ4OTc5NS8zMTE1MzUxODctY2Y5NDYwMmItZmUwYS00MDYxLThiNWYtNDIyMjhmMzVjYTJiLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMTAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzEwVDE4MjMzMlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTQ3Y2M3MzQ5MmU5YWY0OTc0MWJlZThiZmNlOWZiZjkyYzRkOGVhYmJhNzhlZGJlNDc2MGI1OTNkYzA5ZmNhZmYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.k0mWbysVqzs_37LvnW2LIOw4qh5zX79lR3D--IqYHlI)

Ah!, i forgot to remove them!

Waakul commented:
Actually, The overall colors in light mode were cringe XD, just changed it

Samq64 commented on webpages/settings/style.css:
```diff
@@ -147,7 +154,7 @@ h1 {
 #confirmImport,
 #applyLanguageSettingsButton {
-  border-color: var(--orange);
+  background: var(--brand-orange);
```
This should have the text set to `--white-text` to fix it in light mode

Waakul replied:
I'm still doing some tweaks pls wait

Waakul replied:
> This should have the text set to `--white-text` to fix it in light mode

wdym?

Samq64 replied:
![image](https://github.com/ScratchAddons/ScratchAddons/assets/81489795/065e259d-1e5b-4cca-ab3e-d02c17159575)

Waakul replied:
> ![image](https://private-user-images.githubusercontent.com/81489795/311536383-065e259d-1e5b-4cca-ab3e-d02c17159575.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTAwOTY0MzcsIm5iZiI6MTcxMDA5NjEzNywicGF0aCI6Ii84MTQ4OTc5NS8zMTE1MzYzODMtMDY1ZTI1OWQtMWU1Yi00Y2NhLWFiM2UtZDAyYzE3MTU5NTc1LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMTAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzEwVDE4NDIxN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTdhMjZjYTA0YjM0MmYxNDI3MDBmNWUwOTdjZTAxMDBmMzE1ZjhjYzg2ZmUxY2E5MzU1Y2M3MThjYTQxODVjNDUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.ambzlGnI1967XPkKK7zL4pwrdct5JENHsmaN2Y5awuM)

oh yeah

Waakul replied:
Fixed

Samq64 commented:
Do you mind if I clean up the diffs a little?

WorldLanguages commented:
We can't merge any redesigns that get 2 approvals. That general guideline was made with new-addon and code changes PRs in mind.
